stdlib/public/SwiftRemoteMirror/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-add_swift_library(swiftRemoteMirror SHARED TARGET_LIBRARY DONT_EMBED_BITCODE
+add_swift_library(swiftRemoteMirror ${SWIFT_STDLIB_LIBRARY_BUILD_TYPES} TARGET_LIBRARY DONT_EMBED_BITCODE
```

Comment:
@gribozavr You mentioned we always want `libswiftRemoteMirror` to be built as a shared library. Does this mean this should be hardcoded as `SHARED`, and only built when `SWIFT_BUILD_STDLIB` is turned on?